### ISSUE TITLE
Set views of cards like 3rd image

### ISSUE THREAD

**Issue by dipen-solulab** [flschweiger/SwipeStack#10]
How can I set the views of cards like 3rd image?

**Comment by flschweiger** [comment]
Hi @dipen-solulab , currently you only have direct access to the top view. I'll add a method for that in a future release :wink:

**Comment by saintjab** [comment]
@dipen-solulab, if you are talking about how the cards are laid out, check the app:stack_rotation="x". Setting x = 0 will give you that arrangement. Cheers

**Comment by lpraneeth** [comment]
@saintjab yes it worked but it doesn't look like the 3rd Image but the below cards are same width as the top card! I want the bellow cards width little less compared to the TOP layer! as the 3rd image in display

**Comment by lpraneeth** [comment]
![screenshot_2016-11-10-19-47-03 1](https://cloud.githubusercontent.com/assets/22291365/20180052/be2c2a1e-a77e-11e6-8a95-180c1ca9b7b0.png)

**Comment by lpraneeth** [comment]
![screen2](https://cloud.githubusercontent.com/assets/22291365/20180128/2284f626-a77f-11e6-9692-9c1354d14251.png)

I want some thing like this

**Comment by i-m-aman** [comment]
have you find any solution to your problem??
